config/tags.yml

```diff
@@ -11,5 +11,4 @@
 - mentors
 - support
 - teacher training
-- advisers
```

Comment:
@emmaachesongray Can you just add a blog/tag redirect? Suggest going to `/teacher-training-advisers`